### ISSUE TITLE
[Bug]: bvt test error on EKS(s3)

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#6373]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): e23e00f1d01d43e33c509a6d239ccc38b5f93409
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

2022-11-07 17:56:55 ERROR Executor:174 - [ACTUAL RESULT]:
invalid input: insert table is invalid 't1'
2022-11-07 17:56:55 ERROR Executor:171 - [/matrixone/test/distributed/cases/auto_increment/auto_increment_columns.sql][row:427][drop table t1;] was executed failed, con[id=0, user=dump, pwd=111].
2022-11-07 17:56:55 ERROR Executor:173 - [EXPECT RESULT]:
null
2022-11-07 17:56:55 ERROR Executor:174 - [ACTUAL RESULT]:
no such table .t1
2022-11-07 17:56:55 ERROR Executor:171 - [/matrixone/test/distributed/cases/auto_increment/auto_increment_columns.sql][row:429][create table t1 (a int not null auto_increment, b int);] was executed failed, con[id=0, user=dump, pwd=111].
2022-11-07 17:56:55 ERROR Executor:173 - [EXPECT RESULT]:
null
2022-11-07 17:56:55 ERROR Executor:174 - [ACTUAL RESULT]:
not connect to a database
2022-11-07 17:56:55 ERROR Executor:171 - [/matrixone/test/distributed/cases/auto_increment/auto_increment_columns.sql][row:430][insert into t1(b) values (1);] was executed failed, con[id=0, user=dump, pwd=111].
2022-11-07 17:56:55 ERROR Executor:173 - [EXPECT RESULT]:
null
2022-11-07 17:56:55 ERROR Executor:174 - [ACTUAL RESULT]:
invalid input: insert table is invalid 't1'
2022-11-07 17:56:55 ERROR Executor:171 - [/matrixone/test/distributed/cases/auto_increment/auto_increment_columns.sql][row:431][select * from t1;] was executed failed, con[id=0, user=dump, pwd=111].
2022-11-07 17:56:55 ERROR Executor:173 - [EXPECT RESULT]:
a    b
1    1
2022-11-07 17:56:55 ERROR Executor:174 - [ACTUAL RESULT]:
SQL parser error: table "t1" does not exist
2022-11-07 17:56:56 INFO  Executor:191 - The script file[/matrixone/test/distributed/cases/auto_increment/auto_increment_columns.sql] has been executed, and cost: 11.059s, total:237, success:17, failed:220, ignored:0, abnoraml:0
2022-11-07 17:56:56 INFO  Executor:25 - Start to execute the script file[/matrixone/test/distributed/cases/benchmark/tpch/01_DDL/01_create_table.sql] now, and it will take a few moment,pleas wait......
2022-11-07 17:56:57 ERROR Executor:171 - [/matrixone/test/distributed/cases/benchmark/tpch/01_DDL/01_create_table.sql][row:62][CREATE TABLE CUSTOMER(
C_CUSTKEY     INTEGER NOT NULL,
C_NAME        VARCHAR(25) NOT NULL,
C_ADDRESS     VARCHAR(40) NOT NULL,
C_NATIONKEY   INTEGER NOT NULL,
C_PHONE       CHAR(15) NOT NULL,
C_ACCTBAL     DECIMAL(15,2)   NOT NULL,
C_MKTSEGMENT  CHAR(10) NOT NULL,
C_COMMENT     VARCHAR(117) NOT NULL,
PRIMARY KEY (C_CUSTKEY)
);] was executed failed, con[id=0, user=dump, pwd=111].
2022-11-07 17:56:57 ERROR Executor:173 - [EXPECT RESULT]:
null
2022-11-07 17:56:57 ERROR Executor:174 - [ACTUAL RESULT]:
internal error: panic runtime error: index out of range [10] with length 10: 
runtime.goPanicIndex
	/usr/local/go/src/runtime/panic.go:113
github.com/matrixorigin/matrixone/pkg/objectio.(*Block).GetColumn
	/go/src/github.com/matrixorigin/matrixone/pkg/objectio/block.go:66
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.fetchZonemapAndRowsFromBlockInfo
	/go/src/github.com/matrixorigin/matrixone/pkg/vm/engine/disttae/util.go:156
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.genBlockMetas
	/go/src/github.com/matrixorigin/matrixone/pkg/vm/engine/disttae/tools.go:984
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*Transaction).getTableMeta
	/go/src/github.com/matrixorigin/matrixone/pkg/vm/engine/disttae/txn.go:193
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*database).Relation
	/go/src/github.com/matrixorigin/matrixone/pkg/vm/engine/disttae/database.go:66
github.com/matrixorigin/matrixone/pkg/frontend.(*TxnCompilerContext).getRelation
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/session.go:1502
github.com/matrixorigin/matrixone/pkg/frontend.(*TxnCompilerContext).Resolve
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/session.go:1524
github.com/matrixorigin/matrixone/pkg/sql/plan.(*QueryBuilder).buildTable
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/query_builder.go:1670
github.com/matrixorigin/matrixone/pkg/sql/plan.(*QueryBuilder).buildTable
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/query_builder.go:1769
github.com/matrixorigin/matrixone/pkg/sql/plan.(*QueryBuilder).buildJoinTable
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/query_builder.go:1932
github.com/matrixorigin/matrixone/pkg/sql/plan.(*QueryBuilder).buildTable
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/query_builder.go:1755
github.com/matrixorigin/matrixone/pkg/sql/plan.(*QueryBuilder).buildFrom
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/query_builder.go:1557
github.com/matrixorigin/matrixone/pkg/sql/plan.(*QueryBuilder).buildSelect
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/query_builder.go:1092
github.com/matrixorigin/matrixone/pkg/sql/plan.runBuildSelectByBinder
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/build.go:28
github.com/matrixorigin/matrixone/pkg/sql/plan.BuildPlan
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/build.go:56
github.com/matrixorigin/matrixone/pkg/sql/plan.(*BaseOptimizer).Optimize
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/optimize.go:45
github.com/matrixorigin/matrixone/pkg/frontend.buildPlan
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:2215
github.com/matrixorigin/matrixone/pkg/frontend.(*TxnComputationWrapper).Compile
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:2088
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:2711
github.com/matrixorigin/matrixone/pkg/frontend.(*BackgroundHandler).Exec
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/session.go:1863
github.com/matrixorigin/matrixone/pkg/frontend.determineRoleSetHasPrivilegeSet.func1
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/authenticate.go:3469
github.com/matrixorigin/matrixone/pkg/frontend.determineRoleSetHasPrivilegeSet
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/authenticate.go:3507
github.com/matrixorigin/matrixone/pkg/frontend.determineUserHasPrivilegeSet
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/authenticate.go:3631
github.com/matrixorigin/matrixone/pkg/frontend.authenticateUserCanExecuteStatementWithObjectTypeAccountAndDatabase
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/authenticate.go:3999
github.com/matrixorigin/matrixone/pkg/frontend.authenticateUserCanExecuteStatement
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:2312
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:2461
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:3103
github.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:159
runtime.goexit
	/usr/local/go/src/runtime/asm_arm64.s:1172
2022-11-07 17:56:57 ERROR Executor:171 - [/matrixone/test/distributed/cases/benchmark/tpch/01_DDL/01_create_table.sql][row:75][CREATE TABLE ORDERS(
O_ORDERKEY       BIGINT NOT NULL,
O_CUSTKEY        INTEGER NOT NULL,
O_ORDERSTATUS    CHAR(1) NOT NULL,
O_TOTALPRICE     DECIMAL(15,2) NOT NULL,
O_ORDERDATE      DATE NOT NULL,
O_ORDERPRIORITY  CHAR(15) NOT NULL,
O_CLERK          CHAR(15) NOT NULL,
O_SHIPPRIORITY   INTEGER NOT NULL,
O_COMMENT        VARCHAR(79) NOT NULL,
PRIMARY KEY (O_ORDERKEY)
);] was executed failed, con[id=0, user=dump, pwd=111].
2022-11-07 17:56:57 ERROR Executor:173 - [EXPECT RESULT]:
null
2022-11-07 17:56:57 ERROR Executor:174 - [ACTUAL RESULT]:
internal error: panic runtime error: index out of range [10] with length 10: 
runtime.goPanicIndex
	/usr/local/go/src/runtime/panic.go:113
github.com/matrixorigin/matrixone/pkg/objectio.(*Block).GetColumn
	/go/src/github.com/matrixorigin/matrixone/pkg/objectio/block.go:66
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.fetchZonemapAndRowsFromBlockInfo
	/go/src/github.com/matrixorigin/matrixone/pkg/vm/engine/disttae/util.go:156
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.genBlockMetas
	/go/src/github.com/matrixorigin/matrixone/pkg/vm/engine/disttae/tools.go:984
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*Transaction).getTableMeta
	/go/src/github.com/matrixorigin/matrixone/pkg/vm/engine/disttae/txn.go:193
github.com/matrixorigin/matrixone/pkg/vm/engine/disttae.(*database).Relation
	/go/src/github.com/matrixorigin/matrixone/pkg/vm/engine/disttae/database.go:66
github.com/matrixorigin/matrixone/pkg/frontend.(*TxnCompilerContext).getRelation
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/session.go:1502
github.com/matrixorigin/matrixone/pkg/frontend.(*TxnCompilerContext).Resolve
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/session.go:1524
github.com/matrixorigin/matrixone/pkg/sql/plan.(*QueryBuilder).buildTable
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/query_builder.go:1670
github.com/matrixorigin/matrixone/pkg/sql/plan.(*QueryBuilder).buildTable
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/query_builder.go:1769
github.com/matrixorigin/matrixone/pkg/sql/plan.(*QueryBuilder).buildJoinTable
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/query_builder.go:1932
github.com/matrixorigin/matrixone/pkg/sql/plan.(*QueryBuilder).buildTable
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/query_builder.go:1755
github.com/matrixorigin/matrixone/pkg/sql/plan.(*QueryBuilder).buildFrom
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/query_builder.go:1557
github.com/matrixorigin/matrixone/pkg/sql/plan.(*QueryBuilder).buildSelect
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/query_builder.go:1092
github.com/matrixorigin/matrixone/pkg/sql/plan.runBuildSelectByBinder
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/build.go:28
github.com/matrixorigin/matrixone/pkg/sql/plan.BuildPlan
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/build.go:56
github.com/matrixorigin/matrixone/pkg/sql/plan.(*BaseOptimizer).Optimize
	/go/src/github.com/matrixorigin/matrixone/pkg/sql/plan/optimize.go:45
github.com/matrixorigin/matrixone/pkg/frontend.buildPlan
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:2215
github.com/matrixorigin/matrixone/pkg/frontend.(*TxnComputationWrapper).Compile
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:2088
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:2711
github.com/matrixorigin/matrixone/pkg/frontend.(*BackgroundHandler).Exec
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/session.go:1863
github.com/matrixorigin/matrixone/pkg/frontend.determineRoleSetHasPrivilegeSet.func1
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/authenticate.go:3469
github.com/matrixorigin/matrixone/pkg/frontend.determineRoleSetHasPrivilegeSet
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/authenticate.go:3507
github.com/matrixorigin/matrixone/pkg/frontend.determineUserHasPrivilegeSet
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/authenticate.go:3631
github.com/matrixorigin/matrixone/pkg/frontend.authenticateUserCanExecuteStatementWithObjectTypeAccountAndDatabase
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/authenticate.go:3999
github.com/matrixorigin/matrixone/pkg/frontend.authenticateUserCanExecuteStatement
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:2312
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).doComQuery
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:2461
github.com/matrixorigin/matrixone/pkg/frontend.(*MysqlCmdExecutor).ExecRequest
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/mysql_cmd_executor.go:3103
github.com/matrixorigin/matrixone/pkg/frontend.(*Routine).Loop
	/go/src/github.com/matrixorigin/matrixone/pkg/frontend/routine.go:159
runtime.goexit
	/usr/local/go/src/runtime/asm_arm64.s:1172

### Additional information

[cn-20211108.log](https://github.com/matrixorigin/matrixone/files/9954340/cn-20211108.log)

**Comment by fengttt** [comment]
@LeftHandCold  panic in blockmeta, zonemap code.

**Comment by LeftHandCold** [comment]
same as #6524 #6533

**Comment by sukki37** [comment]
@aressu1985 Did you run bvt on a fresh mo cluster?

**Comment by aressu1985** [comment]
> @aressu1985 Did you run bvt on a fresh mo cluster?

yes, but I can not repro this, try again ,and if also not repo, I will close this

**Comment by aressu1985** [comment]
will test this issue last time tomorrow

**Comment by aressu1985** [comment]
fixed:8f3aa6a337b415b7f99339dcc1f730c8c165ef40
